src/screens/InboundReceiveDetail/index.tsx

```diff
@@ -158,19 +135,86 @@ const InboundReceiveDetail: React.FC = () => {
           ]
         }
       ]
-    };
+    }),
+    []
+  );
+
+  const createCompleteReceivingPayload = useCallback(
+    (responseData: any) => ({
+      isShipmentFromPurchaseOrder: responseData.isShipmentFromPurchaseOrder,
+      receiptStatus: 'COMPLETED',
+      containers: responseData.containers,
+      dateShipped: responseData.dateShipped,
+      description: responseData.description,
+      destination: {
```

Comment:
I think you could just have `destination: <id>`, you don't have to send back the name